elixir/.tool-versions

```diff
@@ -1,5 +1,6 @@
 # These are used for the dev environment.
 # This should match the versions used in the built product.
-nodejs 22.20.0
-elixir 1.18.4-otp-27
-erlang 27.3.4.1
+nodejs       22.20.0
+elixir       1.18.4-otp-27
+erlang       27.3.4.1
+chromedriver 144.0.7534.0
```

Comment:
```suggestion
chromedriver latest
```

Unfortunately chromedriver is a bit messy. It wants to be the same version as the actual Chrome install on the macOS host.

I use Chrome (due to required Hubspot extensions). This might work better.

I would also be in favor of dropping the selenium tests since they provide little value at this point and are generally brittle.

Wdyt @bmanifold?